### DISCUSSION TITLE
Welcome to Syncia Discussions!

### DISCUSSION THREAD

**Discussion by Royal-lobster** [Royal-lobster/ChatDockX#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GordanY** [comment]
Hi, I have been testing different plugins for a whole day, and I finally found this amazing plugin. After some use, I would like to make a few suggestions: 
1. A function to copy and paste my customized prompt to other installations. (not cloud sync, just a simple copy-and-paste by text) 
2. A way to change the model directly on the sidebar. For me, I set GPT-3.5 as the default model (since it is much cheaper), however, there are moments when I want to use GPT-4 instead for better output. 

Thank you for making this amazing plugin! :)

**Reply by Royal-lobster** [reply]
Oh yes 😅 its on my top 2 issues as well, i'll work it out this weekend

**Reply by Royal-lobster** [reply]
Released a new version (would take like 4 days to publish to chrome store cuz google reviews), now it will allow you to change model from within sidebar.

About copy pasting your custom prompts to other browsers, if you are using chrome browser, it would by default sync your settings provided you logged in with same google account in both installations, along with that, i'll also work on making settings export to a file and import the settings back with same file
